macros/sql/weekdays_between_dates.sql

```diff
@@ -0,0 +1,31 @@
+--code source: https://discourse.looker.com/t/how-to-count-only-weekdays-between-two-dates/3345
+{%- macro weekdays_between_dates(start, stop) -%}
+(
+date_diff('day', {{start}}, {{stop}}) -
+(
+    (
+        floor (
+            date_diff('day', {{start}}, {{stop}}) / 7
+        ) * 2
+    )
+    + case
+        when
+            date_part(dow, {{start}}) - date_part(dow, {{stop}}) in (1, 2, 3, 4, 5) and
```

Comment:
As far as I can tell `date_part` doesn't work with BigQuery, so for maximum compatibility we could use `extract` instead.

However, since I'm not recommending handling BigQuery compatibility for `datediff` for now, I think it's fine to do the same here and leave it as `date_part` for now.  Though for clarity I think `dayofweek` is preferable to `dow`.

Reply:
Kept date_part for now. From the thread in the Looker article, it looks like someone added a BigQuery version that uses DATETIME_DIFF(cast(${end_time} as datetime),cast(${start_time} as datetime), DAY). Not sure how to make database-specific utilities work for us going forward...